installer/windows/agpl-3.0.rtf

```diff
@@ -0,0 +1,277 @@
+{\rtf1\ansi\deff3\adeflang1025
```

Comment:
I'm not sure whether we need to include AGPL here. We're not importing any of their code here.